src/groupeddataframe/groupeddataframe.jl

```diff
@@ -1070,11 +1070,12 @@ function Base.get(gd::GroupedDataFrame, key::GroupKeyTypes, default)
 end
 
 """
-    filter(fun, gdf::GroupedDataFrame)
-    filter(cols => fun, gdf::GroupedDataFrame)
+    filter(fun, gdf::GroupedDataFrame; ungroup::Bool=false)
+    filter(cols => fun, gdf::GroupedDataFrame; ungroup::Bool=false)
 
 Return a new `GroupedDataFrame` containing only groups for which `fun` returns
-`true`.
+`true`. If `ungroup=true` then a data frame that is a result of passing the
+produced `GroupedDataFrame` to a `DataFrame` constructor is returned instead.
```

Comment:
Maybe this (sorry, cannot make suggestions):
```
Return only groups in `gd` for which `fun` returns `true` as a `GroupedDataFrame`
if `ungroup=false` (the default), or as a data frame if `ungroup=true`.

Reply:
OK